internal/service/databaseuser/resource_database_user_test.go

```diff
@@ -582,25 +545,21 @@ func TestAccConfigRSDatabaseUser_importBasic(t *testing.T) {
 
 func TestAccConfigRSDatabaseUser_importX509TypeCustomer(t *testing.T) {
 	var (
-		dbUser       admin.CloudDatabaseUser
-		resourceName = "mongodbatlas_database_user.test"
-		username     = "CN=ellen@example.com,OU=users,DC=example,DC=com"
-		x509Type     = "CUSTOMER"
-		orgID        = os.Getenv("MONGODB_ATLAS_ORG_ID")
-		projectName  = acc.RandomProjectName()
+		projectID = acc.ProjectIDExecution(t)
+		username  = "CN=ellen@example.com,OU=users,DC=example,DC=com"
```

Comment:
Why aren't us using `acc.RandomName()` here? Same in `TestAccConfigRSDatabaseUser_importLDAPAuthType`

Reply:
i have it changed in a commit but haven't pushed yet